mesos/framework/mirror_maker_executor.go

```diff
@@ -121,6 +121,7 @@ func (e *MirrorMakerExecutor) startMirrorMaker() {
 	e.config.SetIntConfig("queue.size", &mmConfig.ChannelSize)
 	e.config.SetStringConfig("whitelist", &mmConfig.Whitelist)
 	e.config.SetStringConfig("blacklist", &mmConfig.Blacklist)
+	e.config.SetStringConfig("metrics.topic", &mmConfig.ProduceMetricsTopic)
```

Comment:
dots again, lets try to make this struct value default so we can change in one place and others using value don't need to change thx